components/gallery/GalleryItemDescription.vue

```diff
@@ -85,14 +85,35 @@
       </div>
 
       <div
-        v-if="nft?.recipient"
-        class="is-flex is-justify-content-space-between">
-        <p>{{ $t('transfers.recipient') }}</p>
-        <nuxt-link
-          :to="`/${urlPrefix}/u/${nft?.recipient}`"
-          class="has-text-link">
-          <Identity ref="identity" :address="nft?.recipient" />
-        </nuxt-link>
+        v-if="recipient"
+        class="recipient is-flex is-justify-content-space-between">
+        <template v-if="Array.isArray(recipient) && recipient.length > 1">
+          <p>{{ $t('transfers.recipients') }}</p>
+          <ul>
+            <li v-for="([addr, percentile], idx) in recipient" :key="addr">
+              {{ idx + 1 }}.
+              <nuxt-link :to="`/${urlPrefix}/u/${addr}`" class="has-text-link">
+                <Identity ref="identity" :address="addr" />
+              </nuxt-link>
+              <span> ({{ percentile }}%) </span>
+            </li>
+          </ul>
+        </template>
+        <template
+          v-else-if="Array.isArray(recipient) && recipient.length === 1">
+          <p>{{ $t('transfers.recipient') }}</p>
+          <nuxt-link
+            :to="`/${urlPrefix}/u/${recipient[0][0]}`"
+            class="has-text-link">
+            <Identity ref="identity" :address="recipient[0][0]" />
+          </nuxt-link>
+        </template>
+        <template v-else>
+          <p>{{ $t('transfers.recipient') }}</p>
```

Comment:
I have the impression that this line is repeated more than once, maybe put it on top/outside if-else

Reply:
not a vue dev, how could this be optimized? all are slightly different here